src/utill/Log.h

```diff
@@ -0,0 +1,43 @@
+#ifndef NONAME_LOG_H
+#define NONAME_LOG_H
+
+#include <iostream>
+#include <fstream>
+#include <string>
+
+
+
+#define ERRORLOG(message) Log::get().error(message, __PRETTY_FUNCTION__)
+#define LOG(message) Log::get().info(message, __PRETTY_FUNCTION__)
+
+enum Mode{ debug, verbose, nolog, error};
```

Comment:
Sorry for delaying once again, but as `debug, verbose, nolog` etc. are constants (enum variants), they should stay uppercase. Just this one thing plz